api/src/app.ts

```diff
@@ -104,7 +104,10 @@ export const build = async (
   void fastify.register(prismaPlugin);
 
   void fastify.register(testRoutes);
-  void fastify.register(auth0Routes, { prefix: '/auth0' });
+  void fastify.register(auth0Routes, { prefix: '/auth' });
+  if (NODE_ENV === 'development' || NODE_ENV === 'test') {
```

Comment:
```suggestion
	// Note [ENV]: Set the FCC_ENABLE_DEV_LOGIN_MODE="true" to enable development mode login. Tests can also set this in their setup phase.
  if (process.env.FCC_ENABLE_DEV_LOGIN_MODE === "true") {
```

Reply:
We can use this format here and everywhere else if you will.

Reply:
Let's discuss this another time. For now I'll add the env var.

Reply:
I implemented what I think is the spirit of what you asked for!